styles/cart.css

```diff
@@ -0,0 +1,83 @@
+body{
+    margin: 0px;
+}
+.box1{ grid-area: header; }
+.box2{ grid-area: box2;
+display: grid;
+grid-template-columns: 1fr 1fr;
+gap: 1rem; }
+.box3 { grid-area: footer; }
+
+.navigation{
+    width: 98.8%;
+    padding: 0.5rem;
+    position: static; 
+    background-color: yellowgreen;
+}
+
+.nav-bar a{
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    position: relative;
+    left: 70rem;
+}
+
+.footer{
+    width: 99.5%;
+    position: static;
+    margin-bottom: 0%;
+    background-color: yellowgreen;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-areas:
+    'header header header header header header'
+    'box2 box2 box2 box2 box2 box2'
+    'footer footer footer footer footer footer';
+  gap: 6rem;
+  margin-bottom: 0%;
+  background-color: var(--success-green);
+}
+
+.responsive-img{
+    width: 100%;
+    max-width: 100%;
+    max-height: 100%;
+    border-radius: 1rem;
+}
+
+.card{
+    width: 100%;
+}
+.card-img{
+    max-width: 50%;
+}
+.card-landscape .card-content{
+    max-width: 50%;
+    margin: 0%;
+    padding: 0.1rem;
+}
+
+.right-txt{
+    text-align: right;
+}
+
+.card-text{
+    margin-bottom: 0%;
+}
+
+.btn{
+    width: 100%;
+    margin-left: 0%;
+    margin-bottom: 0%;
```

Comment:
You can use margin shorthand property here to assign the values . Do checkout on MDN about how to give shorthand margin properties